astroquery/neodys/tests/test_neodys.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import os
+
+from ...utils.testing_tools import MockResponse
+
+from ... import neodys
+
+DATA_FILE = '2018vp1_eq0.txt'
+
+
+def data_path(filename):
+    data_dir = os.path.join(os.path.dirname(__file__), 'data')
+    return os.path.join(data_dir, filename)
+
+
+# monkeypatch replacement request function
+def nonremote_request(self, request_type, url, **kwargs):
+    with open(data_path(DATA_FILE), 'rb') as f:
+        response = MockResponse(content=f.read(), url=url)
+    return response
+
+
+# use a pytest fixture to create a dummy 'requests.get' function,
+# that mocks(monkeypatches) the actual 'requests.get' function:
+@pytest.fixture
+def patch_request(request):
+    try:
+        mp = request.getfixturevalue("monkeypatch")
+    except AttributeError:
+        mp = request.getfuncargvalue("monkeypatch")
+    mp.setattr(neodys.core.NEODySClass, '_request',
+               nonremote_request)
+    return mp
+
+
+def test_neodys_query():
+    res = neodys.core.NEODyS.query_object(
+        "2018VP1", orbital_element_type='eq', epoch_near_present=0)
+    assert res['Equinoctial State Vector'] == [1.5881497559198392,
```

Comment:
Use some float comparison here and below, e.g. `assert_allclose`, as this produces a failure atm.

```
__________________________________________________ test_neodys_query ___________________________________________________

    def test_neodys_query():
        res = neodys.core.NEODyS.query_object(
            "2018VP1", orbital_element_type='eq', epoch_near_present=0)
>       assert res['Equinoctial State Vector'] == [1.5881497559198392,
        -0.037761493287362, 0.428349632624304, 0.018136658553998,
        0.021742631955829, 14.9386830433394]
E       assert [1.5881497559...9386830433043] == [1.5881497559...9386830433394]
E         At index 0 diff: 1.5881497559213003 != 1.5881497559198392
E         Use -v to get the full diff

astroquery/neodys/tests/test_neodys.py:39: AssertionError
```